remoteworkitem/remoteworkitem_test.go

```diff
@@ -0,0 +1,80 @@
+// +build unit
+
+package remoteworkitem
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/almighty/almighty-core/test"
+)
+
+func provideRemoteGithubData() ([]byte, error) {
+	url := "https://api.github.com/repos/almighty/almighty-core/issues/131"
+	response, err := http.Get(url)
```

Comment:
I think "unit" test should not make external HTTP requests. Can we use a mock object?